test/universal_importer_helper_test.rb

```diff
@@ -46,6 +46,33 @@ def setup
     refute enumerator_ran
   end
 
+  test 'extract and transform with bespoke delimiter' do
```

Comment:
Can we have another test where the file encoding is not UTF8, e.g. using the "broken bar" as a delimiter:

```ruby
broken_bar = 166.chr.force_encoding('Windows-1252') #=> "\xA6"
broken_bar.bytes                                    #=> [166]
broken_bar.encode!('UTF-8')                         #=> "¦"
broken_bar.encode!('UTF-8').bytes                   #=> [194, 166]
```

So in the test definition you'd be using the UTF-8 `¦` character, whereas in the file you'd use the single byte `0xA6` (the Windows-1252 encoding of the same character).

This is basically to test that the separator supplied by the mapping is correctly encoded to match the file's encoding before being used in CSV parsing.

Reply:
^ my bad, this isn't the behaviour we currently have, so ignore for now.